flake.nix

```diff
@@ -1,8 +1,13 @@
 {
   inputs = {
-    # nixpkgs - We use nixpkgs as input of `flakebox`, as it locks things like toolchains,
-    #           in versions that are actually tested in flakebox's CI to cross-compile things
-    #           well. This also saves us download and Nix evaluation time.
+    nixpkgs = {
+      url = "github:nixpkgs/nixos-23.11";
+      # We use nixpkgs as input of `flakebox`, as it locks things like
+      # toolchains, in versions that are actually tested in flakebox's CI to
+      # cross-compile things well. This also saves us download and Nix
+      # evaluation time.
+      follows = "flakebox/nixpkgs";
+    };
```

Comment:
What's the difference between using the input from flakebox directly and specifying our own input but specifying that it follows flakebox?

Reply:
The github actions checking fresheness like it better. It might have some flake.lock effects too, but I'm unclear on it.